docker/Makefile

```diff
@@ -2,59 +2,94 @@ VENDOR ?= ArcadeAI
 PROJECT ?= ArcadeAI
 SOURCE ?= https://github.com/ArcadeAI/arcade-ai
 LICENSE ?= MIT
-DESCRIPTION ?= "Arcade AI for LLM Tool Serving"
-REPOSITORY ?= arcadeai/arcade-ai
+DESCRIPTION ?= "Arcade AI Worker for LLM Tool Serving"
+REPOSITORY ?= arcadeai/worker
 ECR_ENDPOINT ?= 471112909428.dkr.ecr.us-east-1.amazonaws.com
+PLATFORM ?= $(shell uname -m)
 
 VERSION ?= 0.1.0.dev0
 COMMIT ?= $(shell git describe --dirty --always --abbrev=15)
 BUILD_DATE ?= $(shell date -u +"%Y-%m-%dT%H:%M:%SZ")
 IMAGE_NAME ?= worker
 PORT ?= 8002
+INSTALL_TOOLKITS ?= true
 
+# If INSTALL_TOOLKITS is true, we are building the worker image with all toolkits
+# Otherwise, we are building the worker image with no toolkits
+ifeq ($(INSTALL_TOOLKITS), true)
+    REPOSITORY := $(REPOSITORY)
+else
+    REPOSITORY := $(REPOSITORY)-base
+endif
+
+VERSION_TAG := $(VERSION)-$(PLATFORM)
 
 .PHONY: docker-build
 docker-build: ## Build the Docker container
-	@echo "🛠️ Building Docker image ($(VERSION)).."
+	@echo "🛠️ Building Docker image ($(VERSION_TAG)).."
 	@echo "- Commit: $(COMMIT)"
 	@echo "- Build Date: $(BUILD_DATE)"
-	@docker build --build-arg PORT=$(PORT) -f Dockerfile -t $(REPOSITORY):$(VERSION) .. \
+	@docker build --build-arg PORT=$(PORT) -f Dockerfile -t $(REPOSITORY):$(VERSION_TAG) \
+    --build-arg INSTALL_TOOLKITS=$(INSTALL_TOOLKITS) \
 	--build-arg PORT=$(PORT) \
 	--build-arg VERSION="$(VERSION)" \
 	--build-arg COMMIT="$(COMMIT)" \
 	--build-arg BUILD_DATE="$(BUILD_DATE)" \
 	--label=org.opencontainers.image.vendor="$(VENDOR)" \
 	--label=org.opencontainers.image.title="$(PROJECT)" \
 	--label=org.opencontainers.image.revision="$(COMMIT)" \
-	--label=org.opencontainers.image.version="$(VERSION)" \
+	--label=org.opencontainers.image.version="$(VERSION_TAG)" \
 	--label=org.opencontainers.image.created="$(BUILD_DATE)" \
 	--label=org.opencontainers.image.source="$(SOURCE)" \
 	--label=org.opencontainers.image.licenses="$(LICENSE)" \
-	--label=org.opencontainers.image.description=$(DESCRIPTION)
+	--label=org.opencontainers.image.description=$(DESCRIPTION) \
+     ..
+
 
 .PHONY: docker-run
 docker-run: ## Run the Docker container
 	@echo "\n🚀 Run the container with the following ..."
-	@echo ">>> docker run -d -p $(PORT):$(PORT) $(REPOSITORY):$(VERSION)"
+	@echo ">>> docker run -d -p $(PORT):$(PORT) $(REPOSITORY):$(VERSION_TAG)"
 
 
 .PHONY: publish-ecr
-publish-ecr:
-	@echo "🚚 Pushing the Agent image to ECR.."
-	@docker tag $(REPOSITORY):$(VERSION) $(ECR_ENDPOINT)/$(REPOSITORY):$(VERSION)
-	@echo "- pushing $(ECR_ENDPOINT)/$(REPOSITORY):$(VERSION)"
-	@docker push $(ECR_ENDPOINT)/$(REPOSITORY):$(VERSION)
+publish-ecr: ## Publish the Worker image to ECR
```

Comment:
I had left this uncommented previously to deter people from running this outside of CICD